bin/init.sh

```diff
@@ -8,6 +8,14 @@ rm -f django.db
 
 # run database migrations
 
+if [[ ${DJANGO_LOAD_SAMPLE_DATA:-true} = false ]]; then
+    if [[ -d ${DJANGO_MIGRATIONS_DIR} ]]; then
+        cp ${DJANGO_MIGRATIONS_DIR}/0002_*.py ./benefits/core/migrations/
+    else
+        echo "Warning: DJANGO_MIGRATION_DIR needs to be a directory... not loading any data"
```

Comment:
Also, this is a nit - but this warning isn't necessarily accurate right? We could be loading data that already exists in the `./benefits/core/migrations` aside from the sample.

Reply:
True, I'll remove that last part of the message